org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/SDTTestUtils.scala

```diff
@@ -39,10 +39,11 @@ import org.eclipse.jdt.launching.JavaRuntime
 import org.scalaide.core.compiler.IScalaPresentationCompiler
 import org.scalaide.core.internal.project.ScalaProject
 
-/** Utility functions for setting up test projects.
+/**
+ * Utility functions for setting up test projects.
  *
  */
-object SDTTestUtils extends HasLogger {
+trait SDTTestUtils extends HasLogger {
```

Comment:
Full test frameworks have some other design goals to their code than what we have. Changing it to a trait means that it needs more Scaladoc because it needs to explain why and where it should instantiated.